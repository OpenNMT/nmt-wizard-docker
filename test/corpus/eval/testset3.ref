但知情人士表示，潜在竞购方提出的指示性报价与美国银行的期望不符。
许多人期望DTCC“兑现”他们的头寸，但由于不是会员，他们没有任何索取权，只能被迫个人独自管理自己的交易清算事宜。
外界普遍认为，此举是菲律宾政府的对策之一，旨在应对公众对外国援助项目（尤其是中国提供资金支持的项目）有关的腐败指控的关注。 这些项目在没有竞争性竞标的情况下就指定了供应商和承包商。
这是无法开脱的。
上世纪80年代末、90年代初的储蓄和贷款危机之后，TPG创始人之一庞德文(David Bonderman)参与收购了美国储蓄银行(American Savings Bank)，并在1996年将其出售给华盛顿互惠银行(Washington Mutual)并获利巨大，由此在这个领域一举成名。
在经历了三年强劲增长之后，信达目前的总资产规模为2836亿元人民币，由此可见彼时债务剥离的力度之大。
这将有助于我们留住用户。”
总共有37家外资资产管理公司通过与中国公司组建合资公司并掌控少数股权的形式，在中国从事经营活动。
你会感到有些参议员简直想给戴蒙颁个大奖。”
该地区人口超过6亿，合计国内生产总值(GDP)超过巴西或印度。
塔克韦尔创立的ETF证券公司(ETF Securities)现提供300只交易所交易型产品，它们的投资标的曾是富人的专利，只有财富多到可以直接交易期货的投资者才能接触到。
2006年6月，在新总部召开的一个战略会议上，一些董事会成员质疑该银行的理想是否足够远大。
就连正在竞选美国总统的地产大亨、共和党人唐纳德・特朗普(Donald Trump)都已向对冲基金经理们开了火，把他们称为“文员”，说他们缴纳的税款太少。
布鲁塞尔方面正介入撼动伦敦金融城的利率操纵丑闻，提议在整个欧盟范围内将操纵市场指数的尝试列为非法，并对伦敦银行间同业拆借利率(Libor)的设定方式展开根本性的审议。
陶冬表示，此举是向进一步放开人民币汇率迈出的一步，最终目标是实现人民币完全自由兑换，尽管这还有很长的路要走。
过去几个月里，有至少两名纽约联储银行自己的员工，表示出愿意披露以往一向被视为应严格保密的监管信息。
花旗发言人香农・贝尔(Shannon Bell)表示对此交易“无可奉告”。
在债券市场，2012年巴西公司在美国合计发债筹资433亿美元，创历史最高纪录，2011年则为305亿美元。
在其去年5月公布的年报中，ICAP表示该集团部分部门“收到了一些政府机构披露某些信息的要求，以调查Libor是如何制定的”。
巴克莱资本(Barclays Capital)估计，爱尔兰优先银行债务占到国内生产总值(GDP)的38%，低于比利时、西班牙、英国与荷兰。
华盛顿与华尔街关系密切当然不是新闻，尤其是就高盛来说。
据报道对永亨银行感兴趣的有澳大利亚的澳新银行(ANZ)以及新加坡的华侨银行(OCBC)和大华银行(UOB)。
眼下，奥斯本正考虑就此事开展公共调查，这一决定将迫使银行家对这样一个现象作出解释，即： 2000年至2010年这十年的中期，被奥斯本称为“系统性贪婪”的伦敦金融城文化。
欧智华表示，汇丰重点宣传零售分支机构的著名广告词——“全球金融、地方智慧”——现在存在误导作用。
他们在交易及管理市场风险方面都有着直接的工作经验。
据高层监管消息人士称，中国政府还有意降低不断飙升的住房价格，未来数月有可能提高住房抵押贷款的首付比例。
为购买银行不良资产提供财政援助，以及担保对这些资产的部分损失进行补偿。
但童道驰补充表示，在处理不同国家的不同法规方面存在一些司法障碍。
OMFIF计划于5月份在吉隆坡召开类似会议。
对于一个发展中国家而言，民主政治是个奢侈品。
威尔在其自传中表示，普林斯对这项工作日益得心应手。 而帕森斯今年早些时候接受《财富》(Fortune)杂志采访时表示，普林斯刚刚就任的时候，缺乏足够的运营经验，但他最终成了“我们期望的更为优秀的领导者”。 帕森斯是时代华纳的首席执行官，兼任花旗集团董事。
根据从今年3月至12月有效的第一份合约，只要在纽约交易的油价保持在每桶63.50美元以上，J. Aron就必须每月向深南电支付30万美元。
据瑞信称，全球最富有的人群有40%在亚洲。 瑞信的目标是借助资产管理和投资银行部门加大对富有私人客户的产品销售力度，从亚洲的这一目标客户群体身上赚到更多收入。
这使20%的估计成为相对而言的好消息，即便这只是各银行自己提供的初步数据，中国银行业监管机构今年还将进行更为正式的评估。
香港金管局表示，它正在评估此类行为“如果得到证实”，是否对Hibor的设定产生“重大不利影响”。
其结果是，在香港和新加坡这两个金融中心，今年的奖金发放和裁员动作引起了业内人士的失望。
“与有些地方一样，印度有关同性恋的法律没有变，但社会对待同性恋的态度已经改变。
他说： “没有赶紧在香港注册基金产品的基金公司可能会暂时松一口气。”
根据知情人士的说法，马斯特斯决定不去摩科瑞，不过她并未确定离职后做什么。
此前，由于与美国住房市场崩溃相关的巨额减记导致大量高管离职，员工及客户士气和运营绩效均遭受打击，美林曾经历了两年多的煎熬。
如今，对于美国银行（Bank of America）出售在中国建设银行所持的一半股份，这一近乎冷血的撤退行为，中国的高管们所采取的应对措施，明显已成熟睿智了许多。
美国财政部昨日宣布，禁止美国各家银行与汇业银行(Banco Delta Asia)进行业务往来。 此前有调查发现，这家澳门银行曾帮助朝鲜洗钱。
针对为次级抵押贷款证券提供的剩余120亿美元信用保险，该公司进行了4.54亿美元的价值减记，显示出这项业务仍然具有破坏性。
尼日利亚银行业未来的发展可能对整个非洲大陆具有借鉴意义。
对于投资者而言，最重要的问题之一已变成弄明白他们所投资的基金的开曼董事是谁。
最终，这是某种必输的局面。”
这份未公开的名单中，包含了人们普遍预期的诸多跨国银行：
香港银行同业拆息(HIBOR)周三大幅上涨，提高了小型银行从规模更大的竞争对手那里贷款的难度。
没有几个人在享受长时间的抽烟休息时间。”
瑞银强调称，仍“致力于”与中行的业务关系，及其在中国大陆的其它业务。
意大利中期银行(Mediobanca)分析师克里斯托弗・惠勒(Christopher Wheeler)说： “解决这一问题可能需要200亿至300亿美元。”
该公司曾为多种投资者成功筹资，从中国本土的基金的基金Asia Alternatives，到美国华盛顿州。
网商银行(MYBank)的母公司为阿里巴巴的附属公司——浙江蚂蚁小微金融服务集团有限公司（Zhejiang Ant Small & Micro Financial Services Group，简称“蚂蚁金服”）。 蚂蚁金服是中国6家获得新的银行牌照的民营企业之一。 授予民企银行牌照是旨在推动中国金融体系多元化的一个试点项目的一部分。
据报道，去年大学毕业生的年薪中值为2.05万英镑。
他很有能耐，也常常身体力行。”
在本世纪初格林斯潘还在担任主席时，我曾负责报道美联储，当年与他以这种专业风格长谈的记忆迅速重现于我的脑海。
此举也被视为某种意义上的投降。 在过去6个月，由于投资者慌张撤离，纽约五大投行已消失了三家。
一个大型投行部门的主管表示： “许多银行今年进行了六轮裁员，这把公司主管们吓坏了。
与此同时，爱尔兰等欧洲国家正在艰难地为之前银行纾困造成的公共债务负担进行再融资。
具有百年历史的香港金银业贸易场的这一举措，既体现了香港加大推出人民币计价投资产品的努力，也是试图从内地对黄金的强劲需求中分一杯羹。 中国对黄金的需求仍在增长，2011年第二季度需求同比增长了近40%。
这位官员没有指出具体将需要多少资金。
我们希望与经得起时间考验的公司合作，而不愿与那些只想圈钱的公司打交道。”
他的儿子、时年22岁的李兆会削短自己的朋克发型，开始管理业务。
高盛(Goldman Sachs)旗下私人股本公司、新加坡主权财富基金淡马锡(Temasek)以及中国私人股本集团新天域资本(New Horizons)持有双汇少量股权。
香港、台湾和新加坡也有类似的额度协议。
在德国单方面实行“裸卖空”禁令、引起盟国震惊后，欧洲各领导人急忙出面修补主权债务危机下的团结局面。
结果，投资者经常认为债券和信托产品得到政府的隐性担保，这也对2009年以来中国快速的信贷增长起了推波助澜的作用。
此外，人们对能源公司和其它大宗商品交易机构内部的交易量也感到担心。
中国的信用卡市场曾因欠缺支付基础设施而起步较慢，但近年已开始以更快速度增长。
欧盟委员会还希望超越美国等其它地区对“非审计”工作的限制，如果越过某个界限，四大将被迫把其成员所分拆为独立的审计和咨询业务。
事实上，依赖这些“来得快去得也快”的资金，是5年前大批银行需要纾困的一大原因。
随着基准10年期美国国债收益率蹿升至3%，各大银行的证券投资组合收益较年初的近400亿美元急剧减少。
财团的这个最新变化将令法国兴业银行感到苦恼。 法国兴业银行表示，它牵头的财团成员构成一直十分稳定。
但全球性金融海啸的席卷，令所有投资者始料未及，平安投资的富通也未能幸免，股价大幅下跌。”
中国邮政储蓄银行董事长刘安东在挂牌仪式上表示，通过为农村家庭和企业提供信贷，该银行将促进农村地区的发展。
建立全球航空公司这种大胆而又完全合乎逻辑的构想存在了多久？
据日本《日经产业新闻》(Nikkei business daily)报道，预计瑞穗将筹集约4000亿日元，从而成为日本本财政年度最大的一笔融资交易。
不过，中行上周表示，该行的其他主要外国投资者没有出售所持股份的计划。 中国政府仍持有中行多数股权。
但摩根士丹利(Morgan Stanley)和咨询公司奥纬咨询(Oliver Wyman)在去年的一份报告中写道，在监管新规冲击银行以往的获利方式之时，来自抵押品转换的收入“可能将弥补了大部分损失”。
也就是说，Countrywide和美林事实上已经破产，即便支付1美元购买它们的股权，也很可能付出了过高的代价。
消费者业务部门也将裁减3000至4000名员工，得益于自动化取代人工。
建行表示：
但在金融危机以后，清算所却出现在一场全球监管运动的前沿——这场运动旨在改革衍生品市场、降低大银行带来的风险。
这些银行自称，转型是真实的、永久的。
特里表示： “我们将看到一些人才流失，薪资较高的银行家将离开，而薪资较低的人将被聘用到同样的职位，使这种业务模式在这个具体领域发挥作用。”
自我咨询的大型并购的兴起，和位于加州的风险投资业作为科技公司最重要的单一投资来源的崛起，发生在同一时间，这并非巧合。
巴克莱的过错不可能是孤立事件。
花旗的运作过程加深了这种担心。
美国银行今年1月出售28亿美元建行股份时，曾委任美林(Merrill Lynch)以及瑞银(UBS)安排相关事宜。
此次道歉和近几周来的第二笔纾困贷款，表明凯雷集团试图了解凯雷资本公司(Carlyle Capital Corporation)的危机。 自7月4日上市以来，凯雷资本公司连续遭遇了一系列坏消息的打击。
在欧洲，法兰克福的排名上升5位，超过了卢森堡，而华沙跃升26位至第38名，都柏林上升6位至第46名。
他的第一站将与德国中小型企业(Mittelstand)中的一些领先成员会面——这些家族企业构成了德国经济的支柱。
过去的方式是帮助印度企业走出去，美国企业走进来。
相反，欧洲似乎正开始出现政治障碍。 人们担心，即将颁布的“另类投资基金经理指令”(Alternative Investment Fund Managers directive)，可能会使欧盟27个成员国的投资者难以利用欧盟外的私募股权基金。
但为了实行改革，欧洲的政策制定者已形成了多份提议草案，可这些草案不是效果适得其反，就是根本没有可行性。
“好消息，”
传统银行正在反击，它们敦促实施更严厉的监管，并推出了类似的产品和提高了自己的存款利率，但迄今收效甚微。
胡祖六(Fred Hu)是高盛中国区前主管、中国成立最久、知名度最高的私募股权公司之一春华资本(Primavera Capital)的创始人。 他表示，国有企业拥有一些非常出色、极有才干的经营人才，但这些人缺乏推动变革的动力——不论是财务还是其他动力。
