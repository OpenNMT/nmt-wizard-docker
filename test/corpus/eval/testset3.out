然而，知情人士表示，潜在竞购者提出的指示性报价与美银的预期不符。
许多人期望DTCC能“履行”自己的职位，但他们不是成员，没有任何要求，只是被迫单独管理交易的清算。
此举被广泛视为中国政府努力解决公众对与外国援助项目相关的腐败指控的戒心的一部分，尤其是那些由中国资助的项目。中国在没有竞标的情况下提名供应商和承包商。
那不是借口。
TPG联合创始人戴维・邦德曼在上世纪80年代末和90年代初的储蓄和贷款危机之后帮助他树立了声誉，他于1996年以巨额利润参与了收购美国储蓄银行及其后出售给华盛顿互惠银行。
从这个数字来看，经过3年的强劲增长，中国目前的资产总量为2836亿元人民币。
这将帮助我们留住用户。”
共有37家外国基金管理公司通过与中资集团的合资企业中的少数股权，在中国开展业务。
“你会认为，有些参议员想给他一个奖。”
该地区人口超过6亿，国内生产总值总和超过巴西或印度。
图克韦尔创立的ETF Securities目前提供300多种交易所交易产品，这些产品原本是禁止所有人交易的，但只有那些足够富有的人才能直接交易期货。
2006年6月，在新总行召开的一次战略会议上，一些董事会成员质疑该行是否雄心勃勃。
就连竞选美国总统的共和党地产大亨唐纳德・特朗普也对对冲基金经理大加抨击，称他们是“送纸人”，不缴纳应得税。
欧盟正涉入震动伦敦金融城的利率操纵丑闻，提出一项提议，拟非法操纵欧盟各地的市场指数，并对有关Libor设定规则的基本审查。
他表示，此举是人民币全面自由化的一部分，人民币最终目标是完全可兑换-尽管这距离实现完全可兑换还有很长的路要走。
过去几个月，至少两名员工已表明，他们愿意泄露监管信息。传统上，监管信息被认为是严格保密的：
花旗集团发言人香农・贝尔对这笔交易“没有置评”。
在债市，2012年巴西企业创下了此前所有纪录，在美国上市发行的债券筹资433亿美元，而去年同期为305亿美元。
ICAP在5月份的年报中表示，该集团的部分部门“收到一些政府机构的请求，要求它们提供有关Libor如何设定的调查信息”。
巴克莱资本估计，爱尔兰高级银行债务占国内生产总值的38%，低于比利时、西班牙、英国和荷兰。
当然，华盛顿和华尔街之间紧密的联系并不新鲜，尤其是在高盛方面。
澳大利亚的澳新银行和华侨银行以及新加坡的大华银行等都有报道。
奥斯本正考虑对这起事件展开公开调查，这个法庭可能会迫使银行家解释奥斯本在过去10年中期所称的金融城“系统贪婪”文化。
欧智华表示，汇丰著名的广告口号-“全球本地银行”-聚焦于零售分行，如今已具有误导性。
彼等拥有直接的交易及管理市场风险的职业生涯经验。
高级监管人士称，中国政府还试图压低飙升的住宅地产价格，并可能在未来几个月上调抵押贷款最低存款要求。
向银行购买有毒资产提供财务援助，并提供担保，以弥补这些资产的部分损失。
但他补充称，在处理不同国家的不同规则方面，存在法律障碍。
Omfif计划5月在吉隆坡举行类似的会议。
民主是发展中国家的奢侈品。
威尔在回忆录中说，普林斯一直在“成长为这个职位”，而时代华纳首席执行官、花旗集团董事帕森斯今年早些时候对《财富》杂志表示，普林斯在被任命时没有足够的运营经验，但最终却成为“比我们希望的更好的领导人”。
根据从今年3月到12月的第一份合同，只要纽约交易的油价保持在每桶63.50美元以上，J. aron就必须每月支付这家中国集团30万美元。
瑞信预计，全球最富有的人中，有40%位于亚洲。该行希望通过利用资产管理和投行部门，从这些目标客户中获取更多收入，实际上，该行将加快向富有私人客户的产品分销。
这使得20%的估值相对好消息，即便这只是银行自己在今年银行业监管机构进行更正式评估之前提供的初步数据。
中国央行表示，正在评估这种行为是否“如果得到证实”，是否对香港银行同业拆借利率的设定产生“重大影响”。
这让香港和新加坡金融中心今年的奖金发放和裁员感到失望。
“印度是同性恋相关法律没有改变但社会没有改变的地方之一。
他表示： “那些不急于在香港注册基金产品的基金经理可能会暂时松一口气。”
据一位知情人士透露，马斯特斯决定不去摩科瑞，但尚未决定下一步该怎么做。在备忘录中，她被形容为“把业务打造成了当今行业领先的特许经营”。
美林经历了两年的动荡。此前，美国房地产市场崩盘导致大规模资产减记，导致大量高管离职，令员工和客户士气以及经营业绩受挫。
如今，从美国银行从中国建设银行退出的无情部分来看，中国高管们年纪大了，也更聪明了。
美国财政部昨日禁止美国银行与汇业银行进行交易，此前一项调查得出结论，该澳门银行曾帮助朝鲜洗钱。
AIGFP在其售出的120亿美元次级抵押贷款证券信用保险中，亏损4.54亿美元，表明该业务仍有多严重。
尼日利亚银行的未来发展可能对整个非洲大陆产生影响。
对投资者来说，最关键的因素之一是了解他们基金的开曼董事是谁。
最终，这是一种不双赢的局面。”
这份名单并不公开，它包含许多跨国银行的名字，这些名字将是人们普遍预期的：
香港银行同业拆借利率周三大幅上扬，加大了小银行从规模更大的竞争对手那里获取贷款的难度。
没有一群人在抽长烟。”
瑞银强调，该行“致力于”与中行及其它中国内地业务的关系。
Mediobanca分析师克里斯托弗・惠勒表示： “这感觉像是一笔200亿至300亿美元的发行。”
它一直能够为投资者募资，从一只当地基金-亚洲另类基金-到美国华盛顿州。
由阿里巴巴旗下企业浙江蚂蚁金服旗下的库，持有6张新的银行牌照，其中一张是根据一项试点计划授予民营企业的6张银行牌照，该计划旨在实现中国金融体系多元化。
去年，大学毕业生报告的薪资中值为2.05万英镑。
他很有能力，而且非常亲力亲为。”
2000年代初格林斯潘担任美联储主席时，我曾报道过美联储，我对以这种教授风格与格林斯潘长期交谈的记忆很快又回到了我脑海中。
过去6个月，纽约五大投行中的3家因投资者蜂拥而倒闭，此举也被视为某种投降。
“许多银行今年已经进行了6轮裁员，这让高管们感到恐慌，”一家大型投行部门的负责人表示。
与此同时，爱尔兰等欧洲国家正艰难地为过去银行纾困带来的公共债务负担再融资。
这家百年老交易所将此举作为香港加大投资力度、以人民币计价的举措的一部分，也是利用内地黄金需求旺盛而赚钱的尝试。与2010年相比，中国对黄金的需求仍在增长，截至2011年第二季度，黄金需求已跃升近40%。
这位官员不愿透露需要多少钱。
我们希望与那些经得起时间考验的公司合作，而不是那些只想赚钱的公司。”
22岁的儿子李兆辉剪短了朋克发型，开始经营这家公司。
高盛的私人股本部门、新加坡主权财富基金淡马锡和中国私人股本集团新视野持有少量股份。
香港、台湾和新加坡也有类似的配额协议。
面对主权债务危机，欧洲领导人争相恢复团结。德国单方面禁止裸卖空令盟国失望。
结果，投资者往往把债券和信托产品视为政府的隐性支持，这帮助中国自2009年以来迅速实现了信贷繁荣。
能源公司和其他大宗商品交易公司内部的交易量也令人担忧。
近年来，由于缺乏支付基础设施，中国信用卡市场开局缓慢，增长速度开始加快。
该委员会还希望，如果最大的审计网络将成员公司分割成独立的审计和咨询业务，如果这些业务超过一定的门槛，那么它就会超越美国等其它地区对“非审计”工作的限制。
事实上，依赖易得易得的资金是5年前许多银行需要纾困的一大原因。
随着基准的10年期美国国债收益率飙升至3%，大银行证券投资组合的利润从年初的近400亿美元骤降。
该财团的晚期变动将惹恼兴业银行，该行表示，其自己的集团仍保持坚挺。
遗憾的是，全球金融危机让许多投资者措手不及，富通的股价也未能幸免。”
在就职演说中，中国央行行长刘安东表示，将特别努力向农村家庭和企业提供信贷，以“促进”中国农村的发展。
一家全球航空公司的想法被吹捧为一个奇特但非常合乎逻辑的想法有多久了？
《日经商报》报道称，预计该行将融资约4000亿日元，成为日本本财年规模最大的融资发行。
然而，中国银行本周表示，其他主要外国投资者中没有一家计划出售所持股份。中国银行目前仍持有中国政府的多数股权。
但摩根士丹利和咨询师奥纬咨询去年在一份报告中写道，当新规出台时，抵押品转换的收入“很可能抵消掉大部分损失”，这可能会抑制这些银行过去的一些赚钱行为。
换句话说，Countrywide和美林实际上都是破产机构，甚至支付1美元购入股本，这很可能是过头了。
随着自动化取代人，大约3000至4000名员工也将被从消费行业裁减。
建行表示：
但在金融危机之后，它们已走向全球监管努力的前沿，以改革衍生品市场，降低大银行构成的风险。
银行自己声称，这种转变是真实而持久的。
特里表示： “我们将看到一些波动，高薪银行家将离职，低薪人员将被招聘到同一个职位，以使商业模式在这一特定领域发挥作用。”
自我建议的大型并购的兴起，与总部位于加州的风险投资业崛起为科技公司最重要的单一资金来源，并非巧合。
这不可能是一种孤立的不当行为。
花旗的经营过程加剧了这种担忧。
美银授权去年收购的美林和瑞银安排在1月份出售这28亿美元的部分。
这笔道歉和纾困贷款是凯雷试图为凯雷资本(Carlyle Capital corporation,CCC)的危机划清界限。自7月4日上市以来，凯雷一直遭受着一连串坏消息。
在欧洲，法兰克福上升5位，超过卢森堡，华沙上升26位，第38位，都柏林上升6位，第46位。
他的首站将会见Mittelstand的几位主要成员。 mittelstand是德国经济支柱的家族企业。
这曾帮助印度企业走出国门，美国企业来印度。
相反，政治障碍似乎来自欧洲，人们担心即将出台的“另类投资基金管理公司”指令，可能会让欧盟投资者难以利用位于27个成员国之外的私人股本基金。
但是，出于对改革的渴望，欧洲政策制定者已经提出了从反效果到彻底行不通的提案草案。
“好消息，”
传统银行正在反击，敦促加强监管，推出类似产品，提高存款利率，但迄今收效甚微。
高盛前中国区主管、中国最古老、最知名的私人股本公司之一Primavera的创始人胡祖六表示，国企内部有一些非常优秀、有才华的人，但他们缺乏推动改革的激励-无论是金融还是其他。
